### PR TITLE
Refresh recent job list periodically

### PR DESCRIPTION
**Pull request by syucream**
Enable to auto-refresh recent job list on the header periodically (in this PR, per 1min) to get the latest jobs.

image

### CODE REVIEW

File: frontend/src/utils/Constants.ts [hunk 1]
```diff
@@ -142,3 +142,5 @@ export const JobOperations = {
   ROLE_REGISTER_REFERRAL: 19,
   EXPORT_ENTRY_V2: 20,
 };
+
+export const JobRefreshIntervalMilliSec = 60 * 1000;
```
Comment (syucream) [inline frontend/src/utils/Constants.ts:146]: too heavy?

File: frontend/src/components/Header.tsx [hunk 2]
```diff
@@ -16,10 +16,15 @@ import {
 import { makeStyles } from "@mui/styles";
 import React, { FC, useMemo, useState } from "react";
 import { Link } from "react-router-dom";
-import { useAsync } from "react-use";
+import { useInterval } from "react-use";
```
Comment (hinashi) [inline frontend/src/components/Header.tsx:19]: > useInterval

I didn't know that!

**Review by hinashi — APPROVED** [review]
LGTM!

**Review by userlocalhost — APPROVED** [review]
Thank you for your nice implementation 👍 It must be good for AirOne users.

**Pull request merged**